pkg/cluster/internal/delete/delete.go

```diff
@@ -36,13 +36,14 @@ func Cluster(logger log.Logger, p providers.Provider, name, explicitKubeconfigPa
 	kerr := kubeconfig.Remove(name, explicitKubeconfigPath)
 	if kerr != nil {
 		logger.Errorf("failed to update kubeconfig: %v", kerr)
+		return kerr
+	} else {
+		logger.V(0).Infof("Removed context %q from kubeconfig", name)
```

Comment:
this will tell the user we removed it from the kubeconfig even if the kubeconfig didn't contain the context. the context name is also not just the cluster name, and the context is not the only thing in the kubeconfig.

Reply:
Used context name instead of cluster name.